src/io/jenkins/infra/DockerConfig.groovy

```diff
@@ -0,0 +1,61 @@
+package io.jenkins.infra
+
+
+import java.text.SimpleDateFormat
+import java.util.Date
+
+class DockerConfig implements Serializable {
```

Comment:
```suggestion
class DockerConfig {
```

Reply:
Thanks for the suggestion!

If I understand correctly, the reson of NOT implementing Serializable for this object is because there is no pipeline step involved (while the object `InfraConfig` should implement as it uses the `env` object).

Is it correct?